test/cmdlineTests/yul_stack_opt/output

```diff
@@ -39,167 +39,82 @@ object "object" {
 
 
 Binary representation:
-6001808155806002558060035580600455806005558060065580600755806008558060095580600a5580600b5580600c5580600d55808155806002558060035580600455806005558060065580600755806008558060095580600a5580600b5580600c5580600d5580815550
+600180805560028190556003819055600481905560058190556006819055600781905560088190556009819055600a819055600b819055600c819055600d55
 
 Text representation:
     /* "yul_stack_opt/input.yul":98:99   */
   0x01
-  dup1
-  dup2
     /* "yul_stack_opt/input.yul":129:141   */
-  sstore
-    /* "yul_stack_opt/input.yul":98:99   */
-  dup1
-    /* "yul_stack_opt/input.yul":151:160   */
-  0x02
-    /* "yul_stack_opt/input.yul":144:164   */
-  sstore
-    /* "yul_stack_opt/input.yul":98:99   */
-  dup1
-    /* "yul_stack_opt/input.yul":174:183   */
-  0x03
-    /* "yul_stack_opt/input.yul":167:187   */
-  sstore
-    /* "yul_stack_opt/input.yul":98:99   */
-  dup1
-    /* "yul_stack_opt/input.yul":197:206   */
-  0x04
-    /* "yul_stack_opt/input.yul":190:210   */
-  sstore
-    /* "yul_stack_opt/input.yul":98:99   */
-  dup1
-    /* "yul_stack_opt/input.yul":220:229   */
-  0x05
-    /* "yul_stack_opt/input.yul":213:233   */
-  sstore
-    /* "yul_stack_opt/input.yul":98:99   */
-  dup1
-    /* "yul_stack_opt/input.yul":243:252   */
-  0x06
-    /* "yul_stack_opt/input.yul":236:256   */
-  sstore
-    /* "yul_stack_opt/input.yul":98:99   */
-  dup1
-    /* "yul_stack_opt/input.yul":266:275   */
-  0x07
-    /* "yul_stack_opt/input.yul":259:279   */
-  sstore
-    /* "yul_stack_opt/input.yul":98:99   */
-  dup1
-    /* "yul_stack_opt/input.yul":289:298   */
-  0x08
-    /* "yul_stack_opt/input.yul":282:302   */
-  sstore
-    /* "yul_stack_opt/input.yul":98:99   */
-  dup1
-    /* "yul_stack_opt/input.yul":312:321   */
-  0x09
-    /* "yul_stack_opt/input.yul":305:325   */
-  sstore
-    /* "yul_stack_opt/input.yul":98:99   */
-  dup1
-    /* "yul_stack_opt/input.yul":335:344   */
-  0x0a
-    /* "yul_stack_opt/input.yul":328:348   */
-  sstore
-    /* "yul_stack_opt/input.yul":98:99   */
   dup1
-    /* "yul_stack_opt/input.yul":358:368   */
-  0x0b
-    /* "yul_stack_opt/input.yul":351:372   */
-  sstore
-    /* "yul_stack_opt/input.yul":98:99   */
   dup1
-    /* "yul_stack_opt/input.yul":382:392   */
-  0x0c
-    /* "yul_stack_opt/input.yul":375:396   */
   sstore
-    /* "yul_stack_opt/input.yul":98:99   */
-  dup1
-    /* "yul_stack_opt/input.yul":406:416   */
-  0x0d
-    /* "yul_stack_opt/input.yul":399:420   */
-  sstore
-    /* "yul_stack_opt/input.yul":98:99   */
-  dup1
-  dup2
-    /* "yul_stack_opt/input.yul":129:141   */
-  sstore
-    /* "yul_stack_opt/input.yul":98:99   */
-  dup1
     /* "yul_stack_opt/input.yul":151:160   */
   0x02
     /* "yul_stack_opt/input.yul":144:164   */
+  dup2
```

Comment:
I don't think this is still testing what it used to test.

Reply:
Oh, good catch. https://github.com/ethereum/solidity/pull/10286#discussion_r524285770 and @axic did not say anything there made me think that the removal of `sstore` was intended there.

Anyway, not sure what to do about this one. CLI does not allow disabling the optimizer without disabling the stack optimization. There's also no way to get unoptimized asm output currently. Any suggestions?

Reply:
Test converted to Standard JSON. Also moved to #11604 (which this PR now depends on).